### ISSUE TITLE
Improved instructions

### ISSUE THREAD

**Issue by MikeBlueberry** [akiirui/userscript#24]
Hi, thank you for your userscript. I'm not able to use [your userscript](https://greasyfork.org/en/scripts/416271-play-with-mpv). Perhaps you could clarify instructions better for non-tech people?

Here's what I tried so far:
- installed [userscript](https://greasyfork.org/en/scripts/416271-play-with-mpv)
- downloaded and extracted [mpv handler](https://github.com/akiirui/mpv-handler/blob/main/README.md#manual-installation-1) to C:\yt-dlp
- edited config.toml ([uncommented](https://pastebin.com/FJVgbn1V) mpv and yt-dlp to their paths)
- added mpv and yt-dlp to [system environmental path](https://i.ibb.co/j4MvT6y/Captura-de-ecr-2024-08-12-174110.png).
- on a supported site (YouTube), clicked lower bottom left icon;
- a command line shows up, giving [this error](https://i.ibb.co/pJCdCSH/Captura-de-ecr-2024-08-12-181240.png).

**Comment by MikeBlueberry** [comment]
Thanks for getting back to me.
Unfortunately, it's still not working.

I've reverted back config.tml to its default, since you mentioned it doesn't need to be configured in the environment variables are set up (correctly).

- My environment variables are [set up like this](https://i.ibb.co/BVm1HQG/Captura-de-ecr-2024-08-13-084012.png);
- Which results in [this error](https://i.ibb.co/XjJZ4z2/Captura-de-ecr-2024-08-13-084215.png) when trying to play it from a supported site.
- Adding `\` or not adding it to the path produces the same error.

**Comment by MikeBlueberry** [comment]
Yes, those are the binary locations for mpv and yt-dlp as you posted.

I deleted the environment variables, and used your code in the config.toml. I used a 5 minute video on YouTube to test it, lots of yellow lines (and some red) showed up and I thought it was producing an error, but after a 15 seconds or so (I thought it would be quicker for a short video) it opened MPV. 

So it worked. Thanks for sticking with me.
